<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPickerWidget</class>
 <widget class="QWidget" name="ColorPickerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorPicker</string>
  </property>
  <widget class="ColorPicker" name="colorpicker" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="rgbLineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="maxLength">
    <number>128</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="hsvLineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="maxLength">
    <number>128</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="qcolorLineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="maxLength">
    <number>128</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ColorPreview" name="preview" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbAlphaEnabled">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>93</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Alpha</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPicker</class>
   <extends>QWidget</extends>
   <header>colorpicker.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorPreview</class>
   <extends>QWidget</extends>
   <header>colorpicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
